snapcraft_legacy/plugins/v2/_ros.py

```diff
@@ -135,6 +191,48 @@ def plugin_cli():
     pass
 
 
+def get_installed_dependencies(installed_packages_path: str) -> Set[str]:
```

Comment:
This function should be duplicated in the non-legacy ros2-humble version

Reply:
Yup :+1:, will do.